packages/cli/src/UserManagement/routes/auth.ts

```diff
@@ -30,25 +32,23 @@ export function authenticationMethods(this: N8nApp): void {
 				throw new Error('Password is required to log in');
 			}
 
-			let user: User | undefined;
-			try {
-				user = await Db.collections.User.findOne(
-					{ email },
-					{
-						relations: ['globalRole'],
-					},
-				);
-			} catch (error) {
-				throw new Error('Unable to access database.');
-			}
+			const adUser = await handleLdapLogin(email, password);
```

Comment:
should we check here if there is a local user with this email and if that user is the owner? if yes, we skip the call to ldap, which most likely will fail.

Reply:
Good idea. Just added it.

Reply:
Got idea. Just added it.